### ISSUE TITLE
module 'torch' has no attribute 'frombuffer'

### ISSUE THREAD

**Issue by oblessnoob** [rlaphoenix/VSGAN#19]
python 3.9.7
torch 1.9.1

corecode:
egvsr = EGVSR(clip = video, device='cuda')
egvsr.load(r'D:\winpython\VapourSynth64Portable\pretrained_models\EGVSR_iter420000.pth')
egvsr.apply()
video = egvsr.clip

**Comment by oblessnoob** [comment]
the document said vsgan only need torch 1.6.0, it is a bug in your vsgan

**Comment by oblessnoob** [comment]
you didnt fix the issue and close the issue, why

**Comment by rlaphoenix** [comment]
> the document said vsgan only need torch 1.6.0, it is a bug in your vsgan

Yes it said that for old versions and has since been updated. You still need to use torch 1.10.0 or newer.

> you didnt fix the issue and close the issue, why

I provided you the solution already, update torch to 1.10.0 or newer. The issue is not on my end.
